### DISCUSSION TITLE
Can zigbee2mqtt detect 2 devices with same ZigbeeModel?

### DISCUSSION THREAD

**Discussion by buiducnhat** [Koenkk/zigbee2mqtt#22657] [Devices]
Hello, I'm working on bringing some Schneider Electric devices to zigbee2mqtt. I have created a pull request to `zigbee2mqtt-herdsman-converter` for them.

When developing, I see a small problem is: For example, [E8332SRY800ZB_WE](https://www.se.com/vn/vi/product/E8332SRY800ZB_WE/switch-avataron-complete-product-white/) and [E8332SRY800ZB_WG](https://www.se.com/vn/vi/product/E8332SRY800ZB_WG/switch-avataron-complete-product-wine-gold/) are both the SE 2 Gang switch, and they have totally the same technical configurations, the only thing different is their color of the physical device (like the following images, or you can see at the product's page).
![E8332SRY800ZB_WE](https://github.com/Koenkk/zigbee2mqtt/assets/59013407/cc844684-98b8-4e78-a899-17331e1a10f7)
![E8332SRY800ZB_WG](https://github.com/Koenkk/zigbee2mqtt/assets/59013407/09bdd519-69d0-4dae-9cd3-7b93e11adb2e)

So when the device is identified by `zigbee2mqtt`, the model is missing the postfix (**_WE** or **_WG**).

Can we solve this problem, or I may continue developing with the model same as the ZigbeeModel (E8332SRY800ZB), thank you very much!

**Comment by Koenkk** [comment]
If they use the exact same `modelId` (check the data/database.db), we cannot distinguish them.

**Reply by buiducnhat** [reply]
Thank you, exactly I am in this case. So I think I use the root model (without postfix) for them.
